feed.py

```diff
@@ -1,8 +1,9 @@
 # -*- coding: UTF-8 -*-
-import re
-from xml.dom.minidom import Document
 import datetime
+import re
 from collections import namedtuple
+from xml.dom.minidom import Document
```

Comment:
A particular reason for the reordering?
In recent projects I aim for ordering by module name, in this case:
```py
import datetime
from collections import namedtuple
import re
from xml.dom.minidom import Document
```

Reply:
Sorry, I instructed PyCharm to automatically sort and optimize imports. I didn't double check this.

It was set to group plain imports and `from` imports separately, leading to this ordering. But I can configure it to mix those and sort everything alphabetically.  
If your ordering example was meant to be alphabetical, it contains a small error. It should actually be:
```python
from collections import namedtuple
import datetime
import re
from xml.dom.minidom import Document
```
So, you can choose: Would you like to group plain and `from` imports or mix them? :)

Reply:
Oh, sorry, I mean your ordering. I would like to have that alphabetic sort independently from import and from imports. The difference between both ways is in most cases not relevant and might change from time to time.